library/data/registry-entry-attributes/src/main/java/org/quiltmc/qsl/registry/attribute/mixin/RegistryMixin.java

```diff
@@ -0,0 +1,53 @@
+package org.quiltmc.qsl.registry.attribute.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+
+import org.quiltmc.qsl.registry.attribute.api.RegistryEntryAttributeHolder;
+import org.quiltmc.qsl.registry.attribute.impl.*;
+import net.minecraft.util.registry.Registry;
+
+@Mixin(Registry.class)
```

Comment:
Should this be getting applied to Registry or MutableRegistry?

Reply:
Attributes can be attached to any element without a restriction on how it was registered, so Registry seems correct.